templates/features.html

```diff
@@ -94,82 +149,45 @@
     NO_LONGER_PURSUING: 1000,
   };
 
-  // Fill data via AJAX.
-  var url = location.hostname == 'localhost' ? '/static/js/mockdata.json':
-                                               '/features.json';
-
-  var xhr = new XMLHttpRequest();
-  xhr.open('GET', url);
-  xhr.responseType = 'json';
-  xhr.onloadend = function(e) {
-    featureList.features = e.target.response;
-    featureList.metadata = chromeMetadata; // Feature list needs access to the selected milestone.
-    featureList.search = search;
+  featureList.metadata = chromeMetadata; // Feature list needs access to the selected milestone.
+  featureList.search = search;
 
+  featuresPromise.then(function(features) {
+    featureList.features = features;
     updateCount(featureList.filter(search.value));
     search.disabled = false;
-  };
-  xhr.send();
+  });
```

Comment:
There's no `catch()` here. There's no error handling in the original code, so this isn't a regression, but this might be an opportunity to add in some logic that swapped out the `loading...` with an error message.

The way the `loading...` is implemented with `::after` leaves me unsure as to how to do it cleanly. Maybe... create a `showError()` method on `<chromedash-featurelist>` and use that to swap around the internal styles?